packs/softlayer/actions/create_key_pair.yaml

```diff
@@ -0,0 +1,13 @@
+description: 'Creates a keypair by name'
+enabled: true
+entry_point: run.py
+name: create_keypair
+parameters:
+  action:
+    default: create_key_pair
+    immutable: true
+    type: string
+  name:
```

Comment:
Hm, looking at the code, the value of this argument is supposed to be a node hostname?

Or is it supposed to be a node name? or node.name == hostname?